portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase

```diff
@@ -97,6 +97,68 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-151394. LDAP user can be unassigned from groups through ldap. Blocked by LPS-153030."
+	@priority = "5"
+	test CanRemoveUserFromGroupThroughLDAP {
+		property apacheds.enabled = "true";
+		property portal.upstream = "marcell";
```

Comment:
So it's not blocked anymore by LPS-153030 ;)

Reply:
Exactly!